src/main/java/picard/util/IntervalList/IntervalListScatterMode.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2018 The Broad Institute
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package picard.util.IntervalList;
+
+import htsjdk.samtools.util.IntervalList;
+import org.broadinstitute.barclay.argparser.CommandLineParser;
+
+import java.util.function.Supplier;
+
+/**
+ * An enum to control the creation of the various IntervalListScatter objects
+ */
+public enum IntervalListScatterMode implements CommandLineParser.ClpEnum {
+    /**
+     * A simple scatter approach in which all output intervals have size equal to the total base count of the source list divide by the
+     * scatter count (except for possible variance in the final interval list).
+     */
+    INTERVAL_SUBDIVISION(IntervalListScattererWithSubdivision::new, "Scatter the interval list into similarly sized interval lists " +
+            "(by base count), breaking up intervals as needed.") {
+
+    },
+    /**
+     * A scatter approach that differs from {@link IntervalListScatterMode#INTERVAL_SUBDIVISION} in a few ways.
```

Comment:
You don't need to qualify the member in the javadoc link because it's in scope. e.g. you can write `{@link #INTERVAL_SUBDIVISION}`.

Also the java coding guidelines for javadoc recommend not to link every reference in a javadoc. See `Use in-line links economically` in https://www.oracle.com/technetwork/java/javase/tech/index-137868.html